test/jdk/java/awt/FileDialog/DoubleActionESC.java

```diff
@@ -34,9 +34,11 @@
 import java.awt.event.KeyEvent;
 import java.util.concurrent.CountDownLatch;
 
+import static java.util.concurrent.TimeUnit.SECONDS;
+
 /*
  * @test
- * @bug 5097243
+ * @bug 5097243 8355478
```

Comment:
No need to add bug id. We add the bug id only if there is any product change.

```suggestion
 * @bug 5097243
```